src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/AsyncLazy`1.cs

```diff
@@ -117,6 +117,8 @@ public AsyncLazy(Func<CancellationToken, Task<T>> asynchronousComputeFunction, F
             _cacheResult = cacheResult;
         }
 
+        public bool CacheResult => _cacheResult;
```

Comment:
It feels a bit weird we expose this from the core type versus just expecting LoadingTextAndVersionSource to remember the value...

Reply:
No longer needed.